documentation/tutorials/creating_synthetic_tokens.md

```diff
@@ -0,0 +1,144 @@
+# Creating Synthetic Tokens from Truffle console
+
+This tutorial will show you how to create synthetic tokens from the command line using UMA’s synthetic token template. Before beginning this tutorial, please make sure your environment is set up correctly by following the instructions in the “Prerequisites” section. After completing this section, your environment will: 
+* Clone the UMA repo
+* Run Ganache on localhost port 9545
```

Comment:
Maybe rephrase to "Run a personal Ethereum blockchain locally on port 9545"

Reply:
Good idea - will make this change.